### DISCUSSION TITLE
Patrol Test throwing 'Error: target integration_test/patrol.dart is invalid'

### DISCUSSION THREAD

**Discussion by vishalshettyafero** [leancodepl/patrol#1723] [Q&A]
1. I have followed the steps to setup up Patrol using the link: https://patrol.leancode.co/getting-started
2. I have created a new test called 'patrol.dart' below under 'integration_test'. I wanted to test some native features.

**patrol.dart code:**

void main() {
  IntegrationTestWidgetsFlutterBinding.ensureInitialized();
  final fieldText = TextEditingController();
  group(
      'smoke test for the app',
  () {
    patrolTest(
      'verify that user is about to create a group with no devices',nativeAutomation: true,
          (PatrolTester $) async {
          app.main();
          await $.pumpAndSettle();
          await Future.delayed(const Duration(seconds: 2));
          final addbtn= find.byType(FloatingActionButton);
          await $.tap(addbtn);
          await $.pumpAndSettle();
          await Future.delayed(const Duration(seconds: 2));
          await $.native.pressBack();
           await $.native.pressHome();
           await Future.delayed(const Duration(seconds: 5));
        },
      );
       }
  );
}
  4. Command to exectute the test : patrol test --target integration_test/patrol.dart --flavor development
  5. It throws an 'Error: target integration_test/patrol.dart is invalid'
       What could be the reason as I am able to execute the flutter tests successfully?

**Comment by bartekpacia** [comment]
The test file must end with `_test.dart`.

**Reply by vishalshettyafero** [reply]
Made the changes as suggested- we get the below error after running the "patrol test --target integration_test/patrol_test.dart --flavor development" command

<img width="851" alt="patrol_test" src="https://github.com/leancodepl/patrol/assets/139103759/f0381021-9ca2-42da-b467-c10c2770861b">

**Reply by bartekpacia** [reply]
This doesn't look like a problem with Patrol. Is your app building when you `flutter build`? Do what the message says.

**Comment by kousthubshetty-NUM0076** [comment]
try this --> patrol test --flavor development --target integration_test/patrol.dart
